run.sh

```diff
@@ -84,7 +84,7 @@ POWDR_APC_CANDIDATES_DIR=apcs RUST_LOG="debug" OUTPUT_PATH="metrics.json" VPMM_P
   --num-children-leaf 1 \
   --num-children-internal 3 \
   --apc-cache-dir apc-cache \
-  --apc-setup-name $APC_SETUP_NAME \
+  --apc-setup-name ${APC_SETUP_NAME}_${APC}_${APC_SKIP}_${PGO_TYPE}_${BLOCK_NUMBER} \
```

Comment:
@pacheco Why does this contain the block number? I think we should have a single setup for all ETH blocks, right?

I think instead, this contain the list of blocks that were used for PGO?